src/blocks/expandable/block.json

```diff
@@ -8,13 +8,33 @@
 	"icon": "editor-expand",
 
 	"example": {},
+	"attributes": {
+		"buttonBackgroundColor": {
+			"type": "string"
+		},
+		"buttonTextColor": {
+			"type": "string"
+		},
+		"linkText": {
+			"type": "string",
+			"default": "Show more"
+		},
+		"visibleAmount": {
+			"type": "number",
+			"default": 300
+		},
+		"visibleUnit": {
+			"enum": [ "px", "%" ],
```

Comment:
Maybe "vh" would be useful here, given some of the designs? It looked to me as though there were blocks that should be constrained to one viewport height before expanding, but then expanded to fit the contents on click.